geospaas/base_viewer/views.py

```diff
@@ -24,30 +23,18 @@ def instantiation_and_set_form_defaults(self):
         self.forms = [None] * len(self.form_class)
         # loop for instantiating with defaults
         for counter, element_form_class in enumerate(self.form_class):
-            # similar default values can be set via *elif* at the bottom of this statement
-            if 'time_coverage_start' in element_form_class.Meta.fields:
-                self.forms[counter] = element_form_class(
-                    {'time_coverage_start':
-                     timezone.datetime(2000, 1, 1, tzinfo=timezone.utc).date(),
-                     'time_coverage_end': timezone.now().date(),
-                     })
-            # elif: ######## code here for other default values
-                    ######## code here for other default values
-            else:  # instantiating without default values
-                self.forms[counter] = element_form_class({})
-
+            self.forms[counter] = element_form_class({})  # instantiation
```

Comment:
Lets write it in this way because we have separated two action in two separate lines of 26 and 27 from each other. Thus for more clear code and showing the sequence of these actions, let it be as a for loop, please.

Reply:
Done